### ISSUE TITLE
Error: cannot traverse unknown node type: LeftShiftOp

### ISSUE THREAD

**Issue by jcalfee** [eventualbuddha/decaffeinate#108]
```bash
curl https://raw.githubusercontent.com/cryptonomex/graphene-ui/master/dl/src/chain/serializer_types.coffee | decaffeinate 

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 17527  100 17527    0     0  25809      0 --:--:-- --:--:-- --:--:-- 25775

/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:570
    throw new Error('cannot traverse unknown node type: ' + node.type);
          ^
Error: cannot traverse unknown node type: LeftShiftOp
    at childPropertyNames (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:570:11)
    at traverse (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:470:47)
    at /home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:465:9
    at Array.forEach (native)
    at descend (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:456:32)
    at traverse (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:473:5)
    at /home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:465:9
    at Array.forEach (native)
    at descend (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:456:32)
    at traverse (/home/jcalfee/opt/iojs-v2.0.1-linux-x64/lib/node_modules/decaffeinate/dist/decaffeinate.cjs.js:473:5)
```

**Comment by jcalfee** [comment]
I am running the latest npm via `npm i -g decaffeinate`

**Comment by eventualbuddha** [comment]
Fixed in v1.44.1 (includes support for both left and right shift).
